react-backend/client/src/api/index.js

```diff
@@ -240,15 +241,23 @@ export const CbAdmin = {
     axios.post(
       '/api/cb/pwd/reset',
       {
-        formEmail: email,
+        email,
       },
     ),
 
 };
 
 
+export const ErrorUtils = {
+  getErrorStatus: pathOr(null, ['response', 'status']),
+  getValidationErrors: head(pathOr('Unknown error', ['response', 'data', 'validation'])),
+  errorStatusEquals: (error, status) => equals(ErrorUtils.getErrorStatus(error), status),
```

Comment:
What's the benefit of using this function over `response.status === ...` in components?

Reply:
It decouples the error handling code from the structure of the error object. The object path currently is `error.response.status` because we're using `axios`. If we decide to use some other library, it may decide to attach the status a different way.